config/pitest-suppressions/pitest-filters-suppressions.xml

```diff
@@ -216,6 +216,24 @@
     <lineContent>tagMessageRegexp = null;</lineContent>
   </mutation>
 
+  <mutation unstable="false">
```

Comment:
@romani I thought we weren't merging with new mutations added.

Reply:
This PR was 1 year ago when we agreed to have some.

Reply:
@stoyanK7 , can you send us removal of pitest survival in next PR?

Reply:
Yeah, if I get an idea of how to kill it. I have no clue what can be done.

Reply:
@stoyanK7 , please create issue on resolving pitest survivals that you intoduce, and put it in PR description,
it is red flag for all maintainers, we give you special allowance, to do this later on, to not block this epic PR too much.

Reply:
@romani I added commit `minor: fix pitest` https://github.com/checkstyle/checkstyle/pull/12151/commits/9035fa36935f67d0eda5b3564568bb7ae49681de to kill the mutation. Please let me know if that is okay.

 The only problem is that `getFileText()` has to be public (or even better package) so the amount of times it is called can be tested with mockito. If it stays `private`, I do not see any other way than using PowerMock(but I saw that this project ditched PowerMock a while ago in exchange for real tests). I tried mocking the constructor of `FileText` to ensure it is called only once, but I got to the conclusion that PowerMock has to be used. Maybe I am wrong, I just hit a dead end.

Reply:
Now that I am thinking, `getFileText()` has the exact same implementation as in `SuppressWithPlainTextCommentFilter`. We can move that method in `SuppressionUtil.

Reply:
Updated the commit to https://github.com/checkstyle/checkstyle/pull/12151/commits/383b0791daa4db9c4a5c6c57c2afc6d260fa4c18. Please let me know if you agree with
- the extraction of the method
- the test that kills the mutation
 
I will merge the commit in the big one after approval.

Reply:
Not ideal, you created pure unit test.
We banned all pure unit tests a long time ago and gained a lot of benefits in experience of performance of test execution.

We need test with Inputs in use. Can you archive it by running checkstyle with same file or ....
If killing is not possible by usage of Input files, we need to put comment on text method to explain why pure unit test model is used.

Reply:
Opened at https://github.com/checkstyle/checkstyle/issues/12952